core/templates/dev/head/pages/exploration-editor-page/services/exploration-data.service.ts

```diff
@@ -56,17 +59,8 @@ angular.module('oppia').factory('ExplorationDataService', [
       '/createhandler/resolved_answers/' + explorationId);
     var explorationDraftAutosaveUrl = '';
 
-    // TODO (#7221): Exploration data service gets called out of context.
-    if (ContextService.getPageContext() !== PAGE_CONTEXT.COLLECTION_EDITOR) {
-      if (GLOBALS.can_edit) {
-        explorationDraftAutosaveUrl = (
-          '/createhandler/autosave_draft/' + explorationId);
-      } else if (GLOBALS.can_voiceover) {
-        explorationDraftAutosaveUrl = (
-          '/createhandler/autosave_voiceover_draft/' + explorationId);
-      }
-    }
-
+    explorationDraftAutosaveUrl = (
```

Comment:
Why not just assign it directly in line 60?

Reply:
Done